src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -9365,6 +9365,11 @@ static void VectorizedSmall8(ref T xRef, T y, ref T zRef, ref T dRef, nuint rema
                 if (typeof(T) == typeof(float)) return AdvSimd.FusedMultiplyAdd(addend.AsSingle(), x.AsSingle(), y.AsSingle()).As<float, T>();
             }
 
+            if (AdvSimd.IsSupported)
+            {
+                if (typeof(T) == typeof(double)) return AdvSimd.Arm64.FusedMultiplyAdd(addend.AsDouble(), x.AsDouble(), y.AsDouble()).As<double, T>();
```

Comment:
should this just be in the above if block, like was done for `Fma.IsSupported`?

Reply:
It's actually a typo in the if clause... it needs to be AdvSimd.Arm64.IsSupported instead of AdvSimd.IsSupported, and thus have its own check.

Reply:
Ah right, my eyes completely skimmed over the `.Arm64` in there